### ISSUE TITLE
Migrate Topographica's remaining param tests

### ISSUE THREAD

**Issue by ceball** [holoviz/param#184]
Things like instantiation of all classes + pickle, etc

**Comment by tonyfast** [comment]
what's topographica?!?!?

**Comment by jlstevens** [comment]
[Topographica](http://ioam.github.io/topographica/) is where param, HoloViews and a bunch of other things all started. Here is [some code](https://github.com/ioam/topographica) if you are interested in how it all began.

**Comment by ceball** [comment]
Topographica's test suite was probably a pretty useful "downstream" test of param (because param was part of topographica). 

When we hacked out param we didn't have time to move everything, let alone get all the tests - we weren't even sure exactly how much we would be chopping out to produce the standalone param lib. 

It's likely there are still things tested in topographica that are not tested in param, e.g. I just looked really quickly and saw https://github.com/ioam/topographica/blob/master/topo/tests/unit/testFilename.txt ("Filename" does not seem to appear in param's current test suite). Maybe there are other tests scattered around, inline doctests, and so on. Who knows. Whether or not those tests are any good is another question ;)
